lib/index.js

```diff
@@ -107,7 +107,7 @@ function renderFile (root, obj) {
 }
 
 function renderUrl (obj) {
-  return rest(obj.url).then((res) => { return JSON.parse(res.entity) })
+  return rest(obj.url).then((res) => { return JSON.parse(transformRaw(obj, res.entity)) })
```

Comment:
Just saw this. Yeah that makes sense. I'll give it a shot. My latest commit takes care of that ternary and I've got tests that work for all the methods. Right now I'm just calling the transformRaw function in each method, but I think your suggestion is much more efficient and elegant :)